app/packs/src/apps/mydb/elements/details/NumericInputUnit.js

```diff
@@ -103,35 +103,32 @@ export default function NumericInputUnit(props) {
     setValue(newInput);
   };
 
-  const labelWrap = label ? <Form.Label>{label}</Form.Label> : <Form.Label style={{ paddingTop: '15px' }} />;
-  const bsSize = field === 'flash_point' ? 'small' : null;
-
-  const unitSwitch = (
-    <Button
-      disabled={inputDisabled}
-      active
-      onClick={() => { toggleInput(); }}
-      bsSize={bsSize}
-    >
-      {currentUnit}
-    </Button>
-  );
+  const bsSize = field === 'flash_point' ? 'sm' : null;
 
   return (
     <div className={`numericInputWithUnit_${unit}`}>
-      {labelWrap}
+      {label
+        ? <Form.Label>{label}</Form.Label>
+        : <Form.Label style={{ paddingTop: '15px' }} />}
       <InputGroup>
         <Form.Control
           type="text"
-          bsClass="bs-form--compact form-control"
+          className="bs-form--compact"
```

Comment:
You could replace class "bs-form-compact" with "px-1"
This class has just this settings:
padding-left: 5px !important;
padding-right: 5px !important;

But when you change this the cypress test will fail
/spec/cypress/end_to_end/manage_samples.cy.js:
This test needs a specific class name

Maybe you can add a className for the test?
Or do you have an other idea?